config/leviton/vrcs4.xml

```diff
@@ -0,0 +1,111 @@
+       to association groups 1 to 4 corresponding to the active button.
+       When a scene is assigned, the controller sends scene activations and level change messages
+       that offer richer control of the device's behavior.
+
+       The controller has 8 scene groups:
+         - Groups 1-4 are triggered when buttons 1-4 are pressed when their LED is not lit.
+         - Groups 5-8 are triggered when buttons 1-4 are pressed when their LED is already lit.
+
+       To configure the scenes, send SCENE_CONTROLLER_CONF_SET to associate scenes with
+       each scene group.  The simplest solution is to assign scene groups 1-8 to scenes 1-8
+       so they are in one-to-one correspondence using the following messages:
+
+         2D 01 01 01 00 : Set scene group 1 to scene 1
+         2D 01 02 02 00 : Set scene group 2 to scene 2
+         2D 01 03 03 00 : Set scene group 3 to scene 3
+         2D 01 04 04 00 : Set scene group 4 to scene 4
+         2D 01 05 05 00 : Set scene group 5 to scene 5
+         2D 01 06 06 00 : Set scene group 6 to scene 6
+         2D 01 07 07 00 : Set scene group 7 to scene 7
+         2D 01 08 08 00 : Set scene group 8 to scene 8
+
+       Once this is done, the controller will send SCENE_ACTIVATION_SET to the devices
+       in association groups 1-4 whenever their corresponding scenes are triggered.  To handle
+       these messages centrally, make sure to associate the primary controller (node 1) with
+       association groups 1-4.  The primary controller will then receive messages like this:
+
+         2B 01 S# 00 : Sent by controller when scene S# is triggered
+
+       When you press the up/down level below the scene buttons, the controller will send
+       send SWITCH_MULTILEVEL_START_LEVEL_CHANGE and SWITCH_MULTILEVEL_STOP_LEVEL_CHANGE messages
+       to the association group corresponding to the most recently activated scene.
+
+         26 04 40 00 : Sent by controller when the down lever is pressed
+         26 05       : Sent by controller when the down lever is released
+         26 04 00 00 : Sent by controller when the up lever is pressed
+         26 05       : Sent by controller when the up lever is released
+
+       After a scene change, the controller sometimes sends SCENE_ACTUATOR_CONF_GET messages
+       (2C 02 00) to query the scene configuration of the controlled devices.  The controlled
+       devices should reply with SCENE_ACTUATOR_CONF_REPORT (2C 03 <sceneId> FF 00) to confirm
+       the scene change.  Ignoring the message works too although the controller seems to ask
+       several times after each scene change until it gets the expected reply.
+
+       Scene messages interact with the indicator light change messages described above.
+       Turning on and off button LEDs determines whether the controller considers the
+       corresponding scenes to be active.
+  -->
+  <CommandClass id="38">
+    <State>
```

Comment:
We need to port AfterMark to Compatibility rather than state.

Reply:
That sounds good.  How about I remove this section from this patch for now and reintroduce it once the necessary command classes are implemented?

Alternatively, what would you think of encoding the "after mark" bit as an attribute of the CommandClass element itself because it fundamentally changes how the command class is interpreted?